test/scripts/error_handling_attest_fn_call.txtar

```diff
@@ -0,0 +1,25 @@
+# [execute] function call that is successful
+stdin fn-call.json
+exec bky-as attest-fn-call
+cp stdout result.json
+
+# [check] assert stdout contains success output
+exec jq -c -r '.transitive_attested_function_call.claims.output | @base64d | fromjson' result.json
+stdout '{"Success":true,"Error":"","Value":{"number":42}}'
```

Comment:
Had to flatten the json because I couldn't figure out multi-line `stdout` testscript commands

Reply:
@tahardi for larger strings/files where you are matching the whole string rather than specifying a regex, you can include them as a file in the txtar archive and reference them, e.g. 
```txtar
-- expected-output.json --
{
  "Success": true,
  "Error": "",
  "Value": {
    "number": 42
  }
}
```
and I think you would use the `cmp` command to compare stdout to the file, e.g.
```txtar
cmp stdout expected-output.json
```

Reply:
Thanks! I updated the examples with long output to use an `expected-output.json` file. Do you think I should do the same for every example regardless of output length?